### ISSUE TITLE
CSS Modules have flash of unstyled content before loading

### ISSUE THREAD

**Issue by leerob** [vercel/next.js#20540]
# Bug report

## Describe the bug

When loading a page using CSS Modules, there is an unexpected flash of unstyled CSS.

## To Reproduce

Two deployments, identical source code. Before using `next@9.2.0`, after using `next@9.3.0`.

- Before: https://next-cssm-before.vercel.app/cssm
- After: https://next-cssm-after.vercel.app/cssm

This issue is still in the latest version of Next.js. Both of the deployments above are public, you can see the source code there. This doesn't appear to affect styles with `styled-jsx`.

Other examples with this issue:

- https://demo.vercel.store
- https://demo.vercel.events

## Expected behavior

The styles should not flash.

## Screenshots

![CleanShot 2020-12-28 at 10 07 36](https://user-images.githubusercontent.com/9113740/103227702-99885180-48f4-11eb-8b89-9f61a2cbc9c2.gif)

**Comment by Timer** [comment]
This appears to be a bug with Chrome. It does not reproduce in Firefox or Safari.

See demo: https://css-bug.vercel.app/

Closing in favor of #20681

**Comment by Chochek** [comment]
For the record I am seeing this in Firefox as well. The sample above is really simple so it doesn't happen in Firefox or Safari. Perhaps you would be able to replicate using a more complex example that doesn't load as quickly

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
